dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -79,6 +79,8 @@ export type Position = 'inner' | 'outer' | 'none';
 export type Props = {
 	linkTo: string;
 	format: ArticleFormat;
+	/** The format of the article holding the card */
+	contextFormat?: ArticleFormat;
```

Comment:
This prop lets us know which article design the card belongs to when it's displayed as onwards content. Because currently we have a re-design for onward contents cards which is only applied on Gallery articles. 

Once the redesign is rolled out to all article types, this prop can be removed.